### DISCUSSION TITLE
访问面板提示错误日志

### DISCUSSION THREAD

**Discussion by 223333** [whyour/qinglong#1651] [Q&A]
![微信截图_20221013165723](https://user-images.githubusercontent.com/11299674/195550255-9c3a9bea-aca6-46f6-b35c-9e6028b5b88b.png)

**Comment by whyour** [comment]
这个如果使用的是最新版 ， 一般需要等等或者 手动去服务器看看服务状态 

进容器内执行  pm2 logs panel
